extensions/amp-analytics/0.1/events.js

```diff
@@ -181,6 +181,12 @@ export class AnalyticsEvent {
     this.type = type;
     /** @const */
     this.vars = opt_vars || Object.create(null);
+    if (typeof this.vars.elementHeight === 'undefined') {
```

Comment:
I don't understand this?

Reply:
In testing, I noticed that we get elementWidth and elementHeight on the "visible" trigger, but not on the "click" trigger. That's because event.vars.elementWidth/Height weren't defined when the event was a click. So this was my attempt to supply those vars for click events. But, I'm happy to change/remove if there is a better way or if this shouldn't be done.
Triggers I was testing with - note selector is ":root" for both:
```
          "triggers": {
            "sample_visibility_trigger": {
              "on": "visible",
              "selector": ":root",
              "request": "sample_visibility_request"
            },
            "sample_img_click_trigger": {
              "on": "click",
              "selector": ":root",
              "request": "sample_click_request"
            }
          }
```

Reply:
Also, the /*REVIEW*/ below is because the linter says this may cause a reflow. If it does, then I think it should be removed, but I wasn't sure.

Reply:
I see. Yes it is by design that `elementHeight` and `elementWidth` only come with `visible` trigger. Measuring `element.clientWidth`/`element.clientHeight` is expensive because they cause reflow just like `element.getBoundingClientRect()`.
Is there a solid reason to add `elementWidth` `elementHeight` to click trigger? Otherwise, let's remove the change here.

Reply:
Ok, removing. Thanks for confirming the reflow issue.

Reply:
Done